parking-lot-help-request/README.md

```diff
@@ -0,0 +1,118 @@
+# Parking Lot Help Request
+
+Host-free low latency notification system to alert attendants in a parking lot or facility that customers need assistance. To run this project youself you'll need to:
+
+* [Purchase the necessary hardware and configure it](#hardware).
+* [Configure Notehub](#notehub).
+* [Configure the project's Notecard](#notecard-firmware).
+* [Set up a Twilio account and configure the data to route from Notehub to Twilio](#twilio-firmware).
+## Hardware
+
+The following hardware is required to run the Parking Lot Help Request application.
+
+* [Notecarrier A with pre-soldered headers](https://shop.blues.io/products/carr-al)
+* [Notecard LTE Cat-1](https://shop.blues.io/products/note-wbna-500) (for use in North America)
+* [Sparkfun RGB LED breakout - WS2812B](https://www.sparkfun.com/products/13282)
+* [Twidec 12mm momentary push button with pre-soldered wires](https://www.amazon.com/gp/product/B08JHW8BPV/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&th=1)
+* [LiPo battery](https://www.adafruit.com/product/2011)
+* [Solar panel with JST connectors (any of these are compatible)](https://www.seeedstudio.com/catalogsearch/result/?q=solar%20panels)
+* [Breadboard](https://www.adafruit.com/product/64)
+* [Male to male jumper wires](https://www.adafruit.com/product/758)
+* [Break-away 0.1" pin strip male headers](https://www.adafruit.com/product/392)
+
+### Hardware Assembly 
+
+1. After purchasing your hardware, use the [Notecard and Notecarrier Quickstart documentation](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/) to assemble your Notecarrier A and Notecard - don't worry too much over connecting to Notehub: this document will cover it shortly.
+1. Break off 3 male pins in a strip and solder to each side of the Sparkfun LED breakout. (This [video from Adafruit](https://www.youtube.com/watch?v=Z0joOKaQ43A&ab_channel=AdafruitIndustries) provides a good example of how to solder male pins to a board like the one the LED comes in.)
```

Comment:
Maybe worth having a picture of the breakout with the pins soldered on to drop at this point.

Reply:
Good suggestion